openeogeotrellis/geopysparkdatacube.py

```diff
@@ -2501,6 +2511,10 @@ def _save_stitched_tile_grid(self, spatial_rdd, path, tile_grid, crop_bounds=Non
             return jvm.org.openeo.geotrellis.geotiff.package.saveStitchedTileGrid(spatial_rdd.srdd.rdd(), path,
                                                                                   tile_grid, max_compression)
 
+    def _save_zarr_executers(self,spatial_rdd, path, number_bands):
+        jvm = get_jvm()
+        jvm.org.openeo.geotrellis.zarr.zarrWriter.saveZarr(spatial_rdd,path,number_bands)
```

Comment:
A better way to pass through various arguments from python to jvm, is to create something similar to GTiffOptions. This will allow for much easier extension in the future.